src/org/ods/util/HelmStatusData.groovy

```diff
@@ -0,0 +1,181 @@
+package org.ods.util
+
+import com.cloudbees.groovy.cps.NonCPS
+import groovy.transform.PackageScope
+import groovy.transform.TypeChecked
+
+// Relevant data returned by helm status --show-resources -o json
+@TypeChecked
+class HelmStatusData {
+
+    // release name
+    String name
+    String namespace
+
+    HelmStatusInfoData info
+
+    // version is an integer but we map it to a string.
+    String version
+
+    @SuppressWarnings(['IfStatementBraces'])
```

Comment:
Better to comply with the rules instead of suppressing warnings

Reply:
To my eyes the code looks cleaner in this particular case.